pkg/qtv/protocol_writer.go

```diff
@@ -284,16 +311,42 @@ func (qp *qProtocol) putDelta(msg *netMsgW, entnum int, from *entityState, to *e
 	if from.skinNum != to.skinNum {
 		bits |= uSkin
 	}
-	if from.modelIndex != to.modelIndex {
-		bits |= uModel
-	}
 	if from.frame != to.frame {
 		bits |= uFrame
 	}
 	if from.effects != to.effects {
 		bits |= uEffects
 	}
 
+	if (qp.extFlagsFTE1 & ftePextTrans) != 0 {
+		if to.trans != from.trans {
+			evenMoreBits |= uFteTrans
```

Comment:
As baseline is from 0, and if server is old it's also 0, this shouldn't affect compatibility, and same with colourmod. Only breaks old clients if watching new server.